Tools/FilterTestTool/BiquadFilter.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python3
```

Comment:
Do we really need python3?

Reply:
python2 support is ending in 2019. It is a good idea to start using and testing as much python3 as possible.

Reply:
@peterbarker The CI is failing, that is really strange.

Reply:
@peterbarker I removed the `3` from the shebang line and made imports compatible with python2, but it is still working only under python3